draft-ietf-quic-multipath.md

```diff
@@ -702,14 +702,45 @@ maintaining an order list of packets sent on each path. That ordered
 list can then be used to compute acknowledgement gaps per path in
 Packet Threshold tests.
 
-### ACK Delay and Zero-Length CID Considerations
-
-The ACK Delay field of the ACK frame is relative to the largest
-acknowledged packet number (see {{Section 13.2.5 of QUIC-TRANSPORT}}).
-When using paths with different transmission delays, the reported host
-delay will most of the time relate to the path with the shortest latency.
-To collect ACK delays on all the paths, hosts should rely on time stamps
-as described in {{QUIC-Timestamp}}.
+### RTT Estimation Considerations when SPNS is Used {#ack-delay-and-zero-length-cid-considerations}
+
+When SPNS is in use, accurate RTT estimation requires more careful
+considerations. According to {{QUIC-RECOVERY}}, an endpoint generates an RTT
+sample on receiving an ACK frame that meets the following two conditions: (1)
+the largest acknowledged packet number is newly acknowledged, and (2) at least
+one of the newly acknowledged packets was ack-eliciting. The RTT sample,
+latest_rtt is calculated as the time elapsed since the largest acknowledged
+packet was sent. However, when applying the above algorithm with SPNS, one may
+encounter the following issues: (1) RTT of some paths are not updated timely if
+ACKs are mostly returned from other paths, and (2) ACK frames depend on the
+largest received packet of the connection, not the path, and the resulted RTT
+sample may be the sum of the uplink delay of one path and the downlink delay of
+another path. One solution for accurate RTT measurements is to employ
+time-stamps as described in {{QUIC-Timestamp}}. If one chooses not to use
+time-stamps but wants to get reasonable estimation of RTTs on multiple paths
+with single packet number space, the following practices can be used:
+
+For packet receiver (ACK sender):
+
+* Maintain an ACK threshold and an ACK timer for each path. A path should send
+an ACK when it receives ack-eliciting threshold number of ack-eliciting
+packets (e.g., two). And an ack-eliciting packet must be acknowledged within
```

Comment:
A path should send an ACK _frame_ when it receives ack-eliciting threshold number of ack-eliciting
packets (e.g., two) _on the current path_.